_includes/tutorial.html

```diff
@@ -53,15 +54,15 @@
        >
       <img class="item_example-image-link"
            src="{{ site.baseurl }}/public/thumbnails/{{ component.logo }}">
-      <div> <!-- <div> for centering -->
+      <div>
```

Comment:
Why is this change needed?

Reply:
Just removing an old comment, that is no longer needed I guess?

Reply:
More than just not needed. You can't nest comments (guess how I found out).

```
<!-- start of outer comment
        <!-- start of inner comment
        end of inner comment -->
end of outer comment -->
```

The entire comment ends at `end of inner comment-->`, and the rest is parsed on a "best effort basis" to render a mess.

Reply:
I see. Thx for explaining.